src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,39 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    protected int totalSalary;
+    protected int workingHours;
+    protected int salaryPerHour;
+    protected LocalDate workingDateFrom;
+    protected LocalDate workingDateTo;
+    protected LocalDate workingDay;
```

Comment:
This variables should be local variables. No need to make them fields of the class
```suggestion
```